### PR TITLE
A few small UI fixes

### LINKED ISSUE #3210: "Include Chapters & Files" toggle closes the search window when search results are empty
### What happened?

Clicking the switch results in the search window closing. 
search

### What did you expect?

The search window should stay open and the toggle should switch.

### Kavita Version Number - If you don not see your version number listed, please update Kavita and see if your issue still persists.

0.8.3 - Stable

### What operating system is Kavita being hosted from?

Docker (LSIO Container)

### If the issue is being seen on Desktop, what OS are you running where you see the issue?

Windows

### If the issue is being seen in the UI, what browsers are you seeing the problem on?

Firefox

### If the issue is being seen on Mobile, what OS are you running where you see the issue?

Android

### If the issue is being seen on the Mobile UI, what browsers are you seeing the problem on?

Firefox

### Relevant log output

_No response_

### Additional Notes

_No response_

### LINKED ISSUE #3222: Series "summary" field no longer showing newlines in 0.8.3
### What happened?

After upgrading from 0.8.2 to 0.8.3 formatting is no longer shown in the "summary" field of a Series.

### What did you expect?

 It seems like newline characters ("\n") are not being interpreted in the "summary" field. Old behavior reproduced newlines present in the text, new version ignores them.

For example this manga in 0.8.2 :

Screenshot 2024-09-24 at 00 51 53

Vs the same data shown in 0.8.3 : 

Screenshot 2024-09-24 at 00 52 40

The text is show correctly in the edit form : 

Screenshot 2024-09-24 at 01 07 47

### Kavita Version Number - If you don not see your version number listed, please update Kavita and see if your issue still persists.

0.8.3 - Stable

### What operating system is Kavita being hosted from?

Docker (Dockerhub Container)

### If the issue is being seen on Desktop, what OS are you running where you see the issue?

Mac

### If the issue is being seen in the UI, what browsers are you seeing the problem on?

Chrome

### If the issue is being seen on Mobile, what OS are you running where you see the issue?

None

### If the issue is being seen on the Mobile UI, what browsers are you seeing the problem on?

_No response_

### Relevant log output

_No response_

### Additional Notes

This is just the regular docker image (jvmilazz0/kavita) running on an intel-based NAS.

### PR DESCRIPTION
**Pull request by Fesaa**
# Fixed
- Fixed: Fixed read-more text not displaying newlines correctly in some situations (Fixes #3222)
- Fixed: Fixed the search bar closing when toggling show chapters and files toggle (Fixes #3210)

### CODE REVIEW

File: UI/Web/src/app/shared/read-more/read-more.component.ts [hunk 1]
```diff
@@ -35,6 +35,9 @@ export class ReadMoreComponent implements OnChanges {
   hideToggle: boolean = true;
   isCollapsed: boolean = true;
 
+  get displayText() {
```
Comment (majora2007) [inline UI/Web/src/app/shared/read-more/read-more.component.ts]: Why not just move this replacement down to the code that sets currentText? This avoids a getter which is expensive.

**Comment by majora2007** [comment]
I'd be curious to see what it looks like if the search bar didn't close out. I agree it feels "ok", but not ideal.

File: UI/Web/src/app/shared/read-more/read-more.component.ts (hunk 1 above)
Comment (Fesaa) [reply UI/Web/src/app/shared/read-more/read-more.component.ts]: Don't think I had a reason. I will change it

**Comment by Fesaa** [comment]
Just makes it look like this. After clicking on the part next to the checkbox & its label. Rather than still 100%
image

Everything keeps working, if you type/search again. It becomes 100%, if you toggle the settings etc it does as well

**Pull request merged**